src/Runner.Sdk/BuildConstants.cs

```diff
@@ -1,19 +0,0 @@
-namespace GitHub.Runner.Sdk
```

Comment:
if we remove this, then after you clone the repo for the first time, you will get some error in the IDE about `BuildConstants` doesn't exist. 😢